app/controllers/api/projects_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Api::ProjectsController < ActionController::API
+
+  def show
+    @project = resource
+    @comments = @project.comments
```

Comment:
do you need such variables? why you just dont use `@project.comments` in jbuilder. it can simplify life